src/components/common/PageHeader/index.tsx

```diff
@@ -1,42 +1,30 @@
-import useAddressBook from '@/hooks/useAddressBook'
-import useSafeAddress from '@/hooks/useSafeAddress'
 import { Box, Typography } from '@mui/material'
+import type { BoxProps } from '@mui/material'
 import type { ReactElement } from 'react'
 
 import css from './styles.module.css'
 
-const SIDEBAR_HEADER_HEIGHT = 158
-
 const PageHeader = ({
   title,
-  subtitle,
   action,
+  sx = {},
 }: {
   title: string
-  subtitle: string | ReactElement
   action?: ReactElement
+  sx?: BoxProps['sx']
 }): ReactElement => {
-  const safeAddress = useSafeAddress()
-  const addressBook = useAddressBook()
-  const isNamedSafe = !!addressBook[safeAddress]
-  const nameHeight = isNamedSafe ? 20 : 0
-
   return (
     <Box
       className={css.container}
       sx={{
-        height: `${SIDEBAR_HEADER_HEIGHT + nameHeight}px`,
-        top: `-${76 + nameHeight}px`,
+        height: '88px',
```

Comment:
Why does it have to be a fixed height? Can't you just add a bottom margin to the title?